File: Code Chef/JavaScript/500-599/594 Qualify the round QUALIFY.js
let input = '';
process.stdin.on('data', (data) => input += data);

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.trim().split(' ').map(Number));
    
    return intArray;
};

const isQualified = (targetScore, easyProblems, hardProblems) => {
    const totalScore = (easyProblems * 1) + (hardProblems * 2);
    
    (totalScore >= targetScore)
        ? console.log('Qualify')
        : console.log('NotQualify');
};

process.stdin.on('end', () => {
    const testcases = strToInt(input.trim().split('\n'));
    
    testcases.forEach((testcase) => {
        const [targetScore, easyProblems, hardProblems] = testcase;
        isQualified(targetScore, easyProblems, hardProblems);
    });
});

File: Code Chef/C Programming/500-599/594 Qualify the round QUALIFY.c
#include <stdio.h>
int main() 
{
    int t,x,a,b;
    scanf("%d",&t);
    for(int i=0;i<t;i++)
    {
        scanf("%d%d%d",&x,&a,&b);
        if(x<=(a+(2*b)))
        {
            printf("Qualify\n");
        }
        else
        {
            printf("NotQualify\n");
        }
    }
	return 0;
}